### PR TITLE
Basics Update

### LINKED ISSUE #420: please add in these features from the unistgov
https://github.com/usnistgov/macos_security  

loads of commands that could be added to this guide ! 
 
 new security issue for m1 Mac 
 
 It is possible to put a MacBook Pro with Apple Silicon into DFU mode without requiring a password, according to my understanding. This could potentially allow a hacker with physical access to the device to load malware firmware or wipe the device. In the past, firmware passwords were used on Intel-based Macs to prevent this type of attack. However, it is unclear what measures are currently in place to prevent such an attack on Apple Silicon Macs.

One way an attacker could exploit this vulnerability is by modifying the firmware to insert malicious code and then signing it with a fake Apple signature. The attacker could then put the MacBook Pro into DFU mode and load the malicious firmware onto the device.
It is not clear why there are no safeguards in place to prevent this type of attack on Apple Silicon Macs.

### PR DESCRIPTION
**Pull request by kimg45**
I tried to update the intro and basics section.

- removed older names for macOS as they might be confusing to new Mac users, also clarified that this guide targets Apple silicon Macs
- removed an outdated link about an attack on the developer SDK from 2015 that seemed to only be a small screenshot of a bigger document
- removed mention of BSD as I'm not sure that BSD would be useful against a three letter agency, also I think we should leave the OS choice up to the reader and not recommend a different OS inside the macOS guide
- changed the update part to indicate that updates are now done in the settings and apple doesn't offer newer versions of macOS on their support site that I could find
- changed link to an amazon book about backups to instead link to the Time Machine support page
- removed the part about FOSS software as many parts of the guide recommend non-FOSS software and macOS itself isn't FOSS so it felt like it was trying to scare the reader away
- removed [this](https://www.usenix.org/system/files/1401_08-12_mickens.pdf) link because I'm not really sure what it has to do with recognizing threats
- updated the encryption part to instead recommend the [built-in password manager](https://support.apple.com/105115) for passwords

closes #420

### CODE REVIEW

**Review by beerisgood — COMMENTED** [review]
I would remove the keychain part as it looks like deprecated.
If you open it, macOS ask if "Passwords" isn't the thing you want to open and to be fair keychain's UI is very old and not user friendly.

**Comment by kimg45** [comment]
> I would remove the keychain part as it looks like deprecated. If you open it, macOS ask if "Passwords" isn't the thing you want to open and to be fair keychain's UI is very old and not user friendly.

I think that might be better yeah.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
-This guide is a collection of techniques for improving the security and privacy of a modern Apple Macintosh computer ("MacBook") running a recent version of macOS (formerly known as "OS X").
+This guide is a collection of techniques for improving the security and privacy of [Apple silicon](https://support.apple.com/116943) Mac computers running a recent version of macOS. **Using older Macs leaves you open to [security vulnerabilities](https://github.com/axi0mX/ipwndfu?tab%253Dreadme-ov-file#checkm8) on the hardware level that Apple can't patch**. Apple silicon Macs are the minimum recommendation but as a general rule, newer chips are always more secure.
```
Comment (drduh) [inline README.md:1]: Can we somehow qualify what "older Macs" refers to? AFAIK Apple supports the latest two OS releases, but is it clear what hardware is updated?

**Review by drduh — APPROVED** [review]
Good stuff, just one minor nit.

File: README.md (hunk 1 above)
Comment (kimg45) [reply README.md]: Clarified Intel Macs.

**Pull request merged**